lib/process-services-cloud/src/lib/form/components/widgets/display-rich-text/display-rich-text.widget.ts

```diff
@@ -18,8 +18,8 @@
 /* eslint-disable @angular-eslint/component-selector */
 
 import { Component, OnInit, ViewEncapsulation } from '@angular/core';
-import { FormService } from '../../../services/form.service';
-import { WidgetComponent } from '../widget.component';
+import { FormService } from '../../../../../../../core/src/lib/form/services/form.service';
```

Comment:
incorrect import paths, you probably need to review all the paths as IDE might have messed up a bit